### PR TITLE
Add disabled prop in Button and add DropToAttach icon

### PR DESCRIPTION
**Pull request by rjjlucy**
**Issue #:** 

**Description of changes:**
- Add disabled prop in `Button`
- Add `DropToAttach` icon
- Fix style issue in `PopoverMenu`

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes

2. How did you test these changes?
Run local storybook and check the change

3. If you made changes to the component library, have you provided corresponding documentation changes?
Yes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/components/ui/PopOver/Styled.tsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ export const StyledPopOverMenu = styled.ul`
   padding: 0.5rem 0;
   box-shadow: ${(props) => props.theme.popOver.shadow};
   z-index: ${(props) => props.theme.zIndex.popOver};
+  display: table;
```
Comment (p-foucht) [inline src/components/ui/PopOver/Styled.tsx:19]: Why is a `table` display necessary OOC?

File: src/components/ui/icons/DropToAttach/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <path fill="#075FFF" d="M21.934 24.52c.467-.598 1.216-.876 1.967-.73.724.144 1.292.667 1.48 1.365l3.05 11.279c.189.698-.038 1.436-.592 1.924-.376.332-.846.505-1.324.505-.255 0-.513-.05-.76-.151l-4.436-1.586c-.161-.058-.342-.064-.51-.02l-2.833.768c-1.025.276-2.081-.314-2.353-1.317l-1.364-5.045c-.272-1.008.345-2.056 1.376-2.335l2.826-.764c.167-.045.319-.141.428-.271zm1.676.742c-.072-.014-.323-.043-.497.184l-.021.027-3.055 3.634c-.308.367-.718.627-1.185.754l-2.825.764c-.233.063-.377.285-.32.496l1.364 5.044c.056.205.284.32.514.26l2.833-.766c.204-.055.412-.083.619-.083.267 0 .533.046.787.137l4.469 1.6c.298.123.5-.03.554-.08.074-.065.19-.206.136-.408l-3.05-11.28c-.054-.201-.226-.263-.323-.283zm5.45 1.149c.178-.374.627-.531 1-.352 1.302.624 2.261 1.768 2.632 3.137.37 1.37.117 2.84-.692 4.036-.145.214-.382.33-.622.33-.145 0-.29-.042-.42-.13-.343-.232-.433-.698-.2-1.041.564-.834.742-1.856.486-2.803-.257-.948-.925-1.741-1.833-2.177-.373-.18-.53-.627-.352-1z" transform="translate(-375 -646) translate(295.5 362) translate(80 284) translate(6.5)"/>
+    <g>
+      <text transform="translate(0 84)">
+        <tspan x=".14" y="14">Drop to attach file</tspan>
```
Comment (p-foucht) [inline src/components/ui/icons/DropToAttach/index.tsx:18]: Not totally sure how this is being rendered, but does this need to be configurable?

File: src/components/ui/PopOver/Styled.tsx (hunk 1 above)
Comment (rjjlucy) [reply src/components/ui/PopOver/Styled.tsx:19]: Good question. I answered the same question from Dave. You can check here: https://code.amazon.com/reviews/CR-39859486

Basically speaking, `width: fit-content` does not work in Safari webView, which means it does not work in Mac Desktop Client. `display: table;` is an alternate which works exactly like `width: fit-content` and no hurt to web client.

File: src/components/ui/icons/DropToAttach/index.tsx (hunk 2 above)
Comment (rjjlucy) [reply src/components/ui/icons/DropToAttach/index.tsx:18]: Made a small change and now it looks like this

Here is what it looks like in light theme

Screen Shot 2020-12-03 at 3 30 06 PM

Here is what it looks like in dark theme

Screen Shot 2020-12-03 at 3 45 30 PM

**Review by p-foucht — APPROVED** [review]

**Pull request merged**